src/Sulu/Bundle/AdminBundle/Resources/js/components/ArrowMenu/ItemSection.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import React from 'react';
+import type {ChildrenArray, Element} from 'react';
+import Item from './Item';
+import Section from './Section';
+
+type Props = {
+    children: ChildrenArray<Element<typeof Item>>,
+    title: string,
+    onChange: (value: *) => void,
+    value: ?*,
+    icon?: string,
+};
+
+export default class ItemSection extends React.PureComponent<Props> {
```

Comment:
What do you think about renaming to `ItemGroup`? /cc @danrot

Reply:
Both versions work for me. However, the `Section` component would then probably also have to be renamed, right?

Reply:
At first I thought so, but it doesn't have to be. And a standalone `Group` doesn't make sense to me. By the way my initial thought was triggered by the `<optgroup/>` tag of the `<select/>`. And the term `Section` feels overused. But mybe we should stay with `ItemSection`. It's up to you guys ;-)

Reply:
Having a `Section` and `ItemGroup` feels a bit weird to me... I would try to be consistent here, but if it is called `Group` or `Section` doesn't really matter to me.